Sources/Sentry/SentryHttpTransport.m

```diff
@@ -74,18 +79,39 @@ - (id)initWithOptions:(SentryOptions *)options
         self.discardedEvents = [NSMutableDictionary new];
         [self.envelopeRateLimit setDelegate:self];
         [self.fileManager setDelegate:self];
+        self.reachability = reachability;
 
         [self sendAllCachedEnvelopes];
+
+#if !TARGET_OS_WATCH
+        [self.reachability
+               monitorURL:[NSURL URLWithString:@"https://sentry.io"]
```

Comment:
l: do you think the URL used here matters? Would it be helpful to monitor the DSN?

Reply:
It doesn't really matter, it's not going to check if the host is up, once in a while, or something like that. So no, it's not helpful to monitor the DSN.